controllers/apply-next/form-router.js

```diff
@@ -170,80 +221,54 @@ function initFormRouter(formModel) {
     });
 
     /**
-     * Route: New Application
+     * Start application
+     * Redirect to eligibility checker
      */
-    function renderNewApplication(req, res, data = null, errors = []) {
-        const form = enhanceForm({
-            locale: req.i18n.getLocale(),
-            baseForm: formModel,
-            data: data
-        });
-
-        res.render(path.resolve(__dirname, './views/new'), {
-            title: res.locals.formTitle,
-            csrfToken: req.csrfToken(),
-            fields: form.newApplicationFields,
-            errors: errors
-        });
-    }
+    router.get('/start', function(req, res) {
+        res.redirect(`${req.baseUrl}/eligibility/1`);
+    });
 
-    router
-        .route('/new')
-        .get(function(req, res) {
-            renderNewApplication(req, res);
-        })
-        .post(async (req, res) => {
-            const validationResult = formModel.schema.validate(req.body, {
-                abortEarly: false,
-                stripUnknown: true,
-                escapeHtml: true
+    /**
+     * New application
+     * Create a new blank application and redirect to first step
+     */
+    router.get('/new', async function(req, res) {
```

Comment:
Rather than rendering an interstitial page the `new` route creates a new empty application and proceeds to the first step.